src/test/testOrderBook.java

```diff
@@ -28,6 +30,22 @@ void placeOneAsk() {
 		assertEquals(10.2f, testSubject.bestAskPrice);
 		assertEquals(-1f, testSubject.bestBidPrice); // Should be unaffected
 		assertEquals(1.5f, testSubject.sittingVolume);
+
```

Comment:
Create a new function, don't append to the ones that are already there as a unit test is meant to be a single test of functionality.